samples/MegaDrone/src/main/java/com/example/oboe/megadrone/MainActivity.java

```diff
@@ -0,0 +1,58 @@
+package com.example.oboe.megadrone;
+
+/*
+ * Copyright 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.MotionEvent;
+
+public class MainActivity extends AppCompatActivity {
+
+    // Used to load the 'native-lib' library on application startup.
+    static {
+        System.loadLibrary("megadrone");
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        startEngine();
```

Comment:
maybe startEngine() in onStart() and stopEngine() in onStop().

Reply:
Done. However, I'm getting a segmentation fault (Pixel XL, Android 8.1) when `startEngine()` is called for a 2nd time (after bringing the app out of the background), even though, or maybe because `mStream` has been closed previously. Any ideas?

Reply:
Could you please share the crash stack? Also maybe add some logging to verify your hypothesis about the stream.